core/loadpoint.go

```diff
@@ -1421,19 +1421,19 @@ func (lp *Loadpoint) publishSocAndRange() {
 
 		// vehicle target soc
 		// TODO take vehicle api limits into account
-		targetSoc := 100
+		apiLimitSoc := 100
 		if vs, ok := lp.GetVehicle().(api.SocLimiter); ok {
-			if limit, err := vs.TargetSoc(); err == nil {
-				targetSoc = int(math.Trunc(limit))
-				lp.log.DEBUG.Printf("vehicle soc limit: %.0f%%", limit)
+			if limit, err := vs.GetLimitSoc(); err == nil {
+				apiLimitSoc = int(limit)
+				lp.log.DEBUG.Printf("vehicle soc limit: %d%%", limit)
 				lp.publish(keys.VehicleTargetSoc, limit)
```

Comment:
Dann sollten wir den auch in vehicleLimitSoc umbenennen (+ UI)

Reply:
Eigentlich nicht. Das ist ambiguous ggü dem welches man in evcc im Fahrzeug einstellt. Ok?

Reply:
UI sollten wir natürlich machen

Reply:
war ne größere Änderung, da da auch translations dran hingen. nun ists aber alles sauber. Cool dass man das jetzt bei custom verwenden kann. Macht mein Testen einfacher. Könnte ich auch noch mal in die Demo.yaml aufnehmen.